README.md

```diff
@@ -3,3 +3,6 @@ Suite of apps around our research findings.
 
 ### Finance App
 Allowing financial advisors and their clients to efficiently manange client portfolios
+
+### Old App 
+Include data on public companies
```

Comment:
Please bring old before new like so


### Old App 
Include data on public companies

### Finance App
Allowing financial advisors and their clients to efficiently manange client portfolios

Reply:
I updated it now